queries.yaml

```diff
@@ -650,19 +650,42 @@ post_process:
       start_zoom: 8
       end_zoom: 15
 
+  # no lake labels at zoom 0-3:
+  # https://github.com/tilezen/vector-datasource/issues/1730
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: water
+      start_zoom: 0
+      end_zoom: 4
+      properties: [name]
+      geom_types: [Polygon, MultiPolygon]
+      where: >-
+        kind == 'lake'
+
   # before extracting the label placement points, remove small water areas which
   # are probably too small to label.
   - fn: vectordatasource.transform.drop_properties
     params:
       source_layer: water
-      start_zoom: 0
+      start_zoom: 4
       end_zoom: 16
       properties: [name]
```

Comment:
Does this drop all name related properties (like `name:en`, `name:de`, `name:fr`, & etc)?

Reply:
Seems like we should use: `drop_properties_with_prefix` instead?

Reply:
This only drops the `name`, which is sufficient to prevent a label placement from being generated. We only generate those where there is a `name` property. However, it does leave the `name:*` and other name-like properties on the polygon, which is a waste of space.

I added a function specifically for names (and related variants such as `official_name`) in #1774, and I've re-used that here in 01eeaf1 and added test to make sure we clear out the translations from the original polygon too.